.gitignore

```diff
@@ -1,4 +1,6 @@
 .DS_Store
 .rsynced
 .idea/
-executables
\ No newline at end of file
+executables
+node_modules
+package-lock.json
```

Comment:
Ideally we'd want the lockfile version controlled for stability. I know we have historically not done, was this perhaps a practical issue that may go away with this change?

Reply:
I agree - let's commit `/package-lock.json`

Reply:
Done!